source/js/buyers-guide/bg-main.js

```diff
@@ -1,13 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import ReactGA from "../react-ga-proxy.js";
+import ReactGA from "../common/react-ga-proxy.js";
```

Comment:
Let's put this in the common lib, and then import things as `import { ReactGa, googleAnalytics, ... } from "../common";` rather than importing everything as `common`.